ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -35,26 +35,46 @@ def toProtocolElement(self):
 
 
 class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
-    def __init__(self, parentContainer, localId, randomSeed):
+    def __init__(self, parentContainer, localId, randomSeed,
+                 numAssociations=2):
```

Comment:
there should be another similar variable in `serverconfig` that is used to control `numPhenotypeAssociationSets`